### PR TITLE
Added corrected clause 138

### PR DESCRIPTION
**Pull request by Kirundel**

### CODE REVIEW

File: holywar_rules.yaml [hunk 1]
```diff
@@ -35,3 +35,8 @@
   - id: 27
     name: Статья 27 ТКЧХ. Пояснение к правам кекспертным
     text: Если кексперт неправ смотри статью 26 ТКЧХ.
+  - id: 138
+    name: Статья 138 ТКЧХ. О правах фурри
+    text: Фурри и фуррифилы ограничены в правах. Их единственным правом является получение наказаний согласно ТКЧХ.
```
Comment (urandon) [inline holywar_rules.yaml]:
```suggestion
    text: >
        Фурри и фуррифилы ограничены в правах. Их единственным
        правом является получение наказаний согласно ТКЧХ.
```

File: holywar_rules.yaml [hunk 2]
```diff
@@ -35,3 +35,8 @@
   - id: 27
     name: Статья 27 ТКЧХ. Пояснение к правам кекспертным
     text: Если кексперт неправ смотри статью 26 ТКЧХ.
+  - id: 138
+    name: Статья 138 ТКЧХ. О правах фурри
+    text: Фурри и фуррифилы ограничены в правах. Их единственным правом является получение наказаний согласно ТКЧХ. 
+          
+          Прим. 1. Данная редакция статьи не подлежит исправлению по запросу фуррей и фуррифилов.
```
Comment (urandon) [inline holywar_rules.yaml]:
```suggestion
        Прим. 1. Данная редакция статьи не подлежит исправлению
        по запросу фуррей и фуррифилов. 
```

**Review by urandon — CHANGES_REQUESTED** [review]
It have to reformatted, it is not a correct multiline string.
@ZaMaZaN4iK can we add YAML linters to pull request CI checks?
We should try this https://github.com/marketplace/actions/yaml-lint

**Comment by zamazan4ik** [comment]
> It have to reformatted, it is not a correct multiline string.
> @ZaMaZaN4iK can we add YAML linters to pull request CI checks?
> We should try this https://github.com/marketplace/actions/yaml-lint

I like the idea! Please make another PR for CI

**Review by urandon — APPROVED** [review]
Great contribution! 👍🏻

**Comment by urandon** [comment]
> > It have to reformatted, it is not a correct multiline string.
> > @ZaMaZaN4iK can we add YAML linters to pull request CI checks?
> > We should try this https://github.com/marketplace/actions/yaml-lint
> 
> I like the idea! Please make another PR for CI

Great! Please, look at #4

**Comment by zamazan4ik** [comment]
Please resolve conflicts and fix formatting, if necessary

**Review by Periecle — APPROVED** [review]
Good job, @Kirundel!

**Comment by zamazan4ik** [comment]
Thanks!

**Pull request merged**